Assets/Scripts/UI/Tooltip.cs

```
@@ -0,0 +1,221 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace UI
+{
+    public class Tooltip : MonoBehaviour
+    {
+        [Tooltip("Highlight color when hover over object.")]
+        public Color highlightColor = new Color(0.38f, 0.97f, 0.44f);
+
+        [Tooltip("Fade speed of the color change (slow -> quick)")]
+        public float highlightSpeed = 4f;
+
+        [Tooltip("Show a text over the interacted object.")]
+        public bool showTooltip = true;
```

If you invert this e.g.
`hideTooltip`
You don't need to define a default and it's also better. You want to toggle some extra behavior that deviates from the original/default purpose and hiding the tooltip shouldn't be the default

---

Alright i see what you mean.